### PR TITLE
fix dropdown styling

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: src/DatePicker/DatePicker.es6 [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  _generateDays() {
+    let {monthsNavigated, activeIndex} = this.state;
+
+    const startDay = moment().add(monthsNavigated, 'month').
+                            startOf('month').startOf('week').startOf('day');
+    const endDay = moment().add(monthsNavigated, 'month').
+                          endOf('month').endOf('week').startOf('day');
+    const totalDays = endDay.diff(startDay, 'days') + 1;
+
+    const days = [for (i of Array(totalDays).keys()) i].
```
Comment (evanrs) [inline src/DatePicker/DatePicker.es6]: Clever!

**Comment by evanrs** [comment]
Solid code @learykara definitely a :+1:

File: src/DatePicker/DatePicker.es6 (hunk 1 above)
Comment (diezguerra) [inline src/DatePicker/DatePicker.es6]: what's the difference between `[for (i of Array(totalDays).keys()) i]` and ```Array(totalDays).keys()```?

Comment (learykara) [inline src/DatePicker/DatePicker.es6]: `Array(totalDays).keys()` returns an `ArrayIterator`

Comment (diezguerra) [inline src/DatePicker/DatePicker.es6]: I just figured out that totalDays is just an int, so you are just bending over backwards to build a range. Can you get lodash in scope so you can use `_.range`?

Something slightly less (but still) ugly:

```es6
const days = Array(totalDays).fill(0).map(
    (_, index) => {
        return {
            dateObj: dateObj: moment(startDay).add(index, 'day'),
            dayOfYear: moment(startDay).add(index, 'day').dayOfYear()
        });

Comment (diezguerra) [inline src/DatePicker/DatePicker.es6]: Anyway, if not using the lodash solution, I'd add a comment explaining what the maneuver does and why.

Comment (learykara) [inline src/DatePicker/DatePicker.es6]: will do

**Pull request merged**